CMakeLists.txt

```diff
@@ -1,13 +1,22 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.5)
-
-SET(PROJECT_BASE_NAME doocs-generic-chimeratk)
-
-SET(${PROJECT_NAME}_MAJOR_VERSION 01)
-SET(${PROJECT_NAME}_MINOR_VERSION 00)
-SET(${PROJECT_NAME}_PATCH_VERSION 00)
+cmake_minimum_required(VERSION 3.16)
+
+set(PROJECT_BASE_NAME generic-chimeratk)
```

Comment:
We definitely should only have one of these. Or maybe even have "project()" already be defined here.